ceph.spec.in

```diff
@@ -846,8 +846,8 @@ mkdir -p %{_localstatedir}/run/ceph/
 %{_datadir}/ceph/id_dsa_drop.ceph.com
 %{_datadir}/ceph/id_dsa_drop.ceph.com.pub
 %dir %{_sysconfdir}/ceph/
-%dir %{_localstatedir}/log/ceph/
-%dir %{_datarootdir}/ceph/
+%attr(3770,ceph,ceph) %dir %{_localstatedir}/log/ceph/
+%attr(750,ceph,ceph) %dir %{_datarootdir}/ceph/
```

Comment:
I don't think this dir should be owned by ceph user -- this dir resolves to /usr/share/ceph and it raises some security concerns -- i.e.. someone that hacked a ceph daemon running under ceph user could modify the contents of the directory

Also, generally (look at the contents of /usr/share), files under /usr/share tend to be owned by root.